data/categories/vp_vehicles_parts.yml

```diff
@@ -4828,3 +4829,170 @@
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1
+  name: Buses
+  children:
+  - vp-2-4-1-1
+  - vp-2-4-1-2
+  - vp-2-4-1-3
+  - vp-2-4-1-4
+  - vp-2-4-1-5
+  - vp-2-4-1-6
+  - vp-2-4-1-7
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-1
+  name: City Buses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - floor_configuration
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-2
+  name: Coach buses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-3
+  name: School Buses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - floor_configuration
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-4
+  name: Minibus
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-5
+  name: Double Decker Buses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-6
+  name: Articulated Buses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - floor_configuration
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-1-7
+  name: Trolleybuses
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - floor_configuration
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-2
+  name: Semi-Trucks
+  children:
+  - vp-2-4-2-1
+  - vp-2-4-2-2
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-2-1
+  name: Long Haul Semi-Trucks
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-2-2
+  name: Short Haul Semi-Trucks
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-3
+  name: Taxis
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - bus_design_type
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
+- id: vp-2-4-4
+  name: Delivery Vans
+  children: []
+  attributes:
+  - body_color
+  - body_pattern
+  - fuel_supply
+  - item_condition
+  - upholstery_color
+  - upholstery_pattern
```

Comment:
Same for Vans: we already have categories for `Vans` (electric, non-electric and hybrid).
I advise against creating a separate area to keep categories mutually exclusive.